pkg/didcomm/transport/transport_interface.go

```diff
@@ -23,6 +25,7 @@ type InboundMessageHandler func(payload []byte) error
 // It is typically created by using aries.Context().
 type InboundProvider interface {
 	InboundMessageHandler() InboundMessageHandler
+	PackWallet() wallet.Pack
```

Comment:
Just wondering if we can name this `Pack()` instead of `PackWallet()`. From DIDComm perspective Pack/Unpack terms are well defined and gives context by itself.

@fqutishat @troyronda @llorllale thoughts ?

Reply:
I’m not a fan of PackWallet either.